### PR TITLE
slade-git: init at 2018.01.29

### PR DESCRIPTION
**Pull request by esoeylemez**

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -1897,6 +1897,10 @@ with pkgs;
     wxGTK = wxGTK30;
   };
 
+  slade-git = callPackage ../applications/misc/slade/git.nix {
```
Comment (Mic92) [inline pkgs/top-level/all-packages.nix]: `sladeUnstable` is the naming convention here.

**Comment by Mic92** [comment]
@GrahamcOfBorg build sladeUnstable

**Comment by grahamc** [comment]
@GrahamcOfBorg eval

**Comment by Mic92** [comment]
@GrahamcOfBorg eval

**Pull request merged**